components/script/dom/htmlelement.rs

```diff
@@ -534,7 +534,9 @@ impl HTMLElementMethods for HTMLElement {
         }
 
         // Step 8: If previous is a Text node, then merge with the next text node given previous.
-        previous.map(Self::merge_with_the_next_text_node);
+        if let Some(_previous) = previous {
```

Comment:
A variable beginning with underscore usually means that it is unused. Since we can shadow variable bindings in Rust, I think we should be able to reuse the same variable name here i.e 

```
if let Some(previous) = previous {
    Self::merge_with_the_next_text_node(previous)
}
```

Reply:
oh okay, got it! I'll fix it.